### PR TITLE
JS cleanup

### PR DESCRIPTION
**Pull request by benjamincudi**
Working on getting a new JIRA instance set up to test this all against, but this should be functionally identical while hewing closer to common JS code styles. The indentation, equality checks, and switch to `let/const` should also preemptively fix a bunch of things that ESLint (and maybe Prettier for formatting?) would have complained about, so adopting them should be a lower lift

### CODE REVIEW

**Comment by andrei-m** [comment]
Let's goooo!

**Comment by andrei-m** [comment]
@benjamincudi It's possible to set up Jiragraph with the free version of Jira Cloud with just a few field customizations. I let my previous instance expire, but I can set one up again soon to give this a try!

Hopefully jiragraph was easier to set up and run with the previous backend/frontend refactoring, so that the Go app only serves static JS instead of a mix of JS and templatized HTML.

**Comment by benjamincudi** [comment]
> @benjamincudi It's possible to set up Jiragraph with the free version of Jira Cloud with just a few field customizations. I let my previous instance expire, but I can set one up again soon to give this a try!
> 
> Hopefully jiragraph was easier to set up and run with the previous backend/frontend refactoring, so that the Go app only serves static JS instead of a mix of JS and templatized HTML.

I just set one up actually - let's sync up some time to make sure I have it configured well 👍 I can add you to that workspace if you want to get a common one to test against

File: src/IssueList.jsx [hunk 1]
```diff
@@ -19,11 +19,7 @@ class IssueList extends React.Component {
         if (this.state.issues.length === 0) {
             return <div>no recent issues </div>;
         }
-        var issues = [];
-        for (var i = 0; i < this.state.issues.length; i++) {
-            const e = this.state.issues[i];
-            issues.push(<Issue key={e.key} issueKey={e.key} summary={e.summary} />);
-        }
+        const issues = this.state.issues.map((e) => <Issue key={e.key} issueKey={e.key} summary={e.summary} />);
```
Comment (andrei-m) [inline src/IssueList.jsx:22]: Nice! Thanks for the helpful explanations in the commit messages, too. This'll help me not write it quite so outdated-ly next time :)

**Comment by andrei-m** [comment]
@benjamincudi My testing is looking good. I'll leave this open for another day or so in case you want to try it out with a real Jira instance, then I'll merge tomorrow.

**Comment by andrei-m** [comment]
And thank you so much for the contribution! I learned useful things :)

**Review by andrei-m — APPROVED** [review]

**Comment by benjamincudi** [comment]
image

its working!

**Pull request merged**